recipes-ni/ni-hw-scripts/ni-hw-scripts-crio_1.0.bb

```diff
@@ -0,0 +1,54 @@
+SUMMARY = "Support scripts and utilities common to all NI hardware products"
+DESCRIPTION = "Support scripts and utilities for all NI hardware products which are supported by NI LinuxRT."
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+LICENSE = "MIT"
+SECTION = "base"
+
+DEPENDS += "\
+	update-rc.d-native \
+"
+
+SRC_URI += "\
+	file://init.d/handle_cpld_ip_reset \
+"
+
+S = "${WORKDIR}"
+
+
+do_install () {
+	install -d ${D}${sysconfdir}/init.d/
+
+	install -m 0755 ${S}/init.d/handle_cpld_ip_reset    ${D}${sysconfdir}/init.d
+}
+
+pkg_postinst_${PN} () {
+	if [ -n "$D" ]; then
+		OPT="-r $D"
+	else
+		OPT="-s"
+	fi
+
+	update-rc.d $OPT handle_cpld_ip_reset    start 6 1 3 4 5 .
+}
+
+pkg_postrm_${PN} () {
+	if [ -n "$D" ]; then
+		OPT="-f -r $D"
+	else
+		OPT="-f"
+	fi
+
+	update-rc.d $OPT handle_cpld_ip_reset  remove
+}
+
+
+PACKAGE_ARCH = "all"
+PACKAGES_remove += "${PN}-staticdev ${PN}-dev ${PN}-dbg"
+
+FILES_${PN} += "\
+	${sysconfdir}/init.d/handle_cpld_ip_reset \
+"
+
+RDEPENDS_${PN} += "\
```

Comment:
Should this also RDEPEND on whatever package provides `/usr/local/natinst/bin/ninetcfgutil`?

Reply:
Good catch.

Reply:
Added dependency to ni-netcfgutil.